index.js

```diff
@@ -0,0 +1,204 @@
+document.addEventListener('DOMContentLoaded', () => {
+    setupFormListener()
+})
+
```

Comment:
https://www.loom.com/share/f5addcd490a0413e940b670e43ade662

Reply:
I now realize why you had to add the 'DOMContentLoaded' listener. You didn't add defer to your script tag.